pages/stats/_groupname.vue

```diff
@@ -480,3 +444,42 @@ export default {
   methods: {}
 }
 </script>
+
+<style scoped lang="scss">
+@import 'color-vars';
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins/_breakpoints';
+
+.chart-wrapper {
+  flex-direction: column;
+
+  @include media-breakpoint-up(md) {
+    flex-direction: row;
+  }
+}
+
+.card {
+  &.chart {
+    height: 100%;
+  }
+}
+
+.titlelogo {
+  width: 140px;
+  height: 140px;
+  object-fit: cover;
+}
+
+.purple {
+  color: $color-purple !important;
+}
+
+.gold {
+  color: $color-gold !important;
+}
+
+.green {
+  color: green !important;
```

Comment:
Appreciate this is legacy but should really be a variable.

Reply:
True.  I'll fix that after merging.

Reply:
I thought I'd got all of the colours when I created the variable file but obviously not!